packages/utils/edgevpn/definition.yaml

```diff
@@ -1,7 +1,6 @@
 name: edgevpn
 category: utils
-version: "0.28.3"
-tag: "0.24.6"
```

Comment:
We should keep the `version` one, because the pipeline that builds the packages when the version is bumped, relies on this field. The change should be minimal. Let's just change this to `.Values.version`: https://github.com/kairos-io/packages/blob/608b1d23a802dbe902d0a5f8cdf722d393bd255c/packages/utils/edgevpn/build.yaml#L12